charts/kubermatic/static/store-container-new.yaml

```diff
@@ -8,6 +8,7 @@ command:
 - |
   set -e
   s3cmd \
+    --no-check-certificate \
```

Comment:
So now we configure both a CA, but also disable any certificte checks? What's the point of that then?

Reply:
Customers should put the CA they use for their cert into the global CA bundle as documented (and then blindly deleted) in https://github.com/kubermatic/docs/pull/524